File: tool/verify_frontend_checklists.dart
#!/usr/bin/env dart
/*
 * Frontend checklist guard entrypoint.
 *
 * Run:
 *   dart run tool/verify_frontend_checklists.dart
 *   dart run tool/verify_frontend_checklists.dart --strict
 */

import 'dart:io';

import 'frontend_guard/core.dart';

Future<void> main(List<String> arguments) async {
  final exitCode = await FrontendGuardRunner().run(arguments);
  if (exitCode != 0) {
    exit(exitCode);
  }
}

File: tool/verify_frontend_checklists.py
#!/usr/bin/env python3
"""
Frontend checklist guard entrypoint.

Run:
  python tool/verify_frontend_checklists.py
  python tool/verify_frontend_checklists.py --strict
"""

from frontend_guard.core import main


if __name__ == "__main__":
    raise SystemExit(main())
